src/generators/typescript/presets/utils/ExampleFunction.ts

```diff
@@ -0,0 +1,78 @@
+import { TypeScriptRenderer } from '../../TypeScriptRenderer';
+import { CommonModel } from '../../../../models';
+
+/**
+ * Inferring first acceptable value from the model.
+ * 
+ * @param model 
```

Comment:
If you add params, please also add param for `renderer` :) My last comment, everything is good.